managed/resource_cluster.go

```diff
@@ -270,6 +270,37 @@ and modify the backup schedule of the cluster being created.`,
 							},
 						}),
 					},
+					"azure_cmk_spec": {
```

Comment:
I think you can add a validator to be sure only azure or aws or gcp spec are specified, you have an example in the metrics eexporter.

Reply:
I checked the metrics exporter code, it uses `schemavalidator.ConflictsWith` to ensure that only out of `grafana_spec` or `datadog_spec` be present. But this validator can only be used with two attributes.

The use case here is to ensure that only one of these three - `aws cmk spec`, `gcp cmk spec` or `azure cmk spec` must be present. 

Hence, I have included the logic of custom validation within the code itself. Please TAL.

Reply:
https://github.com/yugabyte/terraform-provider-ybm/blob/7b2e8040766fbaccf45a4e981f88720214d43261/managed/resource_cluster.go#L727

Reply:
@arishta-yb the function can take multiple path.expression according the function sign. `ConflictsWith(attributePaths ...path.Expression)`